test/test_api/test_project_api.py

```diff
@@ -1190,6 +1190,14 @@ def test_task_progress(self):
         res = self.app.get('/api/project/1/taskprogress?Fruit=null', follow_redirects=True, headers=headers)
         assert res.status_code == 200
 
+        # query for the count of all task count using null keyword
+        res = self.app.get('/api/project/1/taskprogress?Fruit=Null', follow_redirects=True, headers=headers)
+        assert res.status_code == 200
+
+        # query for the count of all task count using null keyword
+        res = self.app.get('/api/project/1/taskprogress?Fruit=NULL', follow_redirects=True, headers=headers)
+        assert res.status_code == 200
```

Comment:
Should the tests assert on the expected "count" of returned tasks? Currently, it appears that only the status code is being checked, but not the actual data being returned. Thank you.

Reply:
It could also be useful to include a test-case for an unknown fruit type.

```python
res = self.app.get('/api/project/1/taskprogress?Fruit=INVALID', follow_redirects=True, headers=headers)
# assert the result count is 0
```

Reply:
Okay, I can make these changes. Originally I was planning to deploy today, but at this moment I don't think that I'll be able to deploy today. Is it okay if this is delayed to Monday?